app/assets/stylesheets/_skeleton.sass

```diff
@@ -26,6 +26,17 @@ body
       width: 30%
       display: inline-block
       text-align: right
+  ul.user-icons
```

Comment:
@wlffann - this may conflict with the PR I just made. I moved all this stuff to _user_partial.sass and I also worked on the images